src/main/java/pitt/search/semanticvectors/LuceneUtils.java

```diff
@@ -40,24 +40,14 @@
 import java.io.IOException;
 import java.nio.file.FileSystems;
 import java.util.ArrayList;
-import java.util.Hashtable;
 import java.util.List;
-import java.util.Map;
 import java.util.TreeSet;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.logging.Logger;
 
 import org.apache.lucene.document.Document;
-import org.apache.lucene.index.LeafReader;
```

Comment:
Typically we've avoided "import *". (Partly because it's been beaten out of me by being banned in some companies.)

I don't mind accepting it but might change it later.